kernel/kernel-api/src/main/java/io/delta/kernel/internal/data/TransactionStateRow.java

```diff
@@ -133,4 +135,15 @@ public static List<String> getPartitionColumnsList(Row transactionState) {
   public static String getTablePath(Row transactionState) {
     return transactionState.getString(COL_NAME_TO_ORDINAL.get("tablePath"));
   }
+
+  /**
+   * Get the maxRetries from scan state {@link Row} returned by {@link
```

Comment:
Is it a "scan state Row"? I thought it is a "transaction state Row"?

Reply:
oops